staging/cse/windows/configfunc.ps1

```diff
@@ -63,7 +63,18 @@ function Adjust-PageFileSize()
 {
     Logs-To-Event -TaskName "AKS.WindowsCSE.AdjustPageFileSize" -TaskMessage "Start to adjust pagefile size"
 
-    wmic pagefileset set InitialSize=8096,MaximumSize=8096
+    try {

```

Comment:
Not a blocker: should the new code be enabled only for WS2015?
```
if (2015) { new }
else { old }
```

Reply:
These powershell commands work in all Window OS version AKS supports so I think that we should not differentiate them.